new-lamassu-admin/src/pages/Wallet/helper.js

```diff
@@ -15,9 +15,10 @@ const WalletSchema = Yup.object().shape({
   zeroConf: Yup.string().required(),
   zeroConfLimit: Yup.number()
     .integer()
-    .required()
+    .required('Zero Conf Limit is a required field')
     .min(0)
     .max(CURRENCY_MAX)
+    .transform(value => (isNaN(value) ? 0 : value))
```

Comment:
There should be a `transformNumber` in the codebase. Can this 0 be swapped to null? Also isn't the bug because of how zeroConfLimit used to be optional and is now required? I think we can just change the logic in the strip row function.